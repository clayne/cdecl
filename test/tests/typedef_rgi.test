EXPECTED_EXIT=65
cdecl <<END
typedef register int SI
END
