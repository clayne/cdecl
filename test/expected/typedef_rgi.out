typedef register int SI
        ^
9: error: "typedef register int" is illegal for built-in type
